tests/ignite/contrib/handlers/test_tensorboard_logger.py

```diff
@@ -32,6 +33,15 @@ def test_optimizer_params_handler_wrong_setup():
         handler(mock_engine, mock_logger, Events.ITERATION_STARTED)
 
 
+class TestTensorboardLogger(unittest.TestCase):
+    @patch("ignite.contrib.handlers.TensorboardLogger")
+    def test_add_scalar(self, mock_writer):
+        # Create a TensorboardLogger instance
+        logger = TensorboardLogger()
+        # Call the add_scalar method
+        logger.add_scalar("loss", 0.5)
```

Comment:
We need to check that `logger.writer.add_scalar` was called with `("loss", 0.5)`